config.json

```diff
@@ -318,6 +312,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 7
+      },
+      {
+        "slug": "all-your-base",
+        "name": "All Your Base",
+        "uuid": "bb51a364-e9de-4e98-bbd9-a8cdd0bc76b3",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5
```

Comment:
Not sure how to rank the difficulty here so feel free to suggest a different ranking! Is there anywhere I can read about how this works? I'm assuming the rankings are out of 10. Also, how do these difficulties translate to the easy, medium, hard tags on the website?

Reply:
The difficulties range from 1 to 9. On the exercism.org website, the difficulty level is only shown as Easy, Medium, Hard. The mapping is:
Easy = 1, 2, 3
Medium = 4, 5, 6
Hard = 7, 8, 9

The exercises show up sorted as in config.json, and we're free to sort however we choose.
I think it's a good idea to sort the exercises by difficulty, and within each difficulty level, sort by name.
I think a difficulty of 5 seems about right for this exercise. 👍

Reply:
Actually, could you please move your exercise to the right place in `config.json`? That would be between the `perfect-numbers` exercise and the `flatten-array` exercise (since we sort by difficulty then name).
Also, could you please run `bin/configlet fmt -u` after you've done that, and type `y` when asked, to ensure that the `config.json` file is formatted as Exercism expects? Thanks!

Reply:
@ageron Sounds good. Updated!